alkos/kernel/CMakeLists.txt

```diff
@@ -14,10 +14,11 @@ include_directories(include)
 include_directories(${SYSROOT}/usr/include)
 include_directories(test)
 
-add_compile_definitions(ALKOS_TEST)
-
 add_compile_definitions(__ALKOS_KERNEL__=1)
 
+# Uncomment to start test framework
```

Comment:
...

Reply:
Will be fixed in cicd pr. Cmake files are not ready yet